<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализатор функций</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QLabel" name="function_text">
          <property name="font">
           <font>
            <family>Source Code Pro Black</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
padding: 2px;
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>Функция не задана</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="createTab">
           <attribute name="title">
            <string>Запись</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вас приветствует анализатор логических функций.&lt;/p&gt;&lt;p&gt;Для начала работы выберите удобный способ задания логической функции:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="buttonTypeManual">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>Записать функцию вручную</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="buttonGenerateFromTable">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>Сгенерировать функцию по таблице истиннсти</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="buttonGenerateFromScheme">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>Перейти в редактор схем</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Преобразования</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;В данной вкладке можно выполнять различные преобразования. Для выполнения преобразования нужно выбрать тип преобразования и нажать на кнопку &amp;quot;Преобразовать&amp;quot;, результат появится в окне вывода. Его можно сохранить как текущую функцию, воспользовавшись кнопкой снизу.&lt;/p&gt;&lt;p&gt;Доступные преобразования:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
              <item>
               <widget class="QComboBox" name="conversion_selector">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>Символьная запись</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Математическая запись</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Юникод-запись</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Английская запись</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Русская запись</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>СКНФ</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>СДНФ</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Конъюктивное упрощение</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Дизъюнктивное упрощение</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_convert">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Преобразовать выражение</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вывод:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="conversion_log">
              <property name="font">
               <font>
                <family>Source Code Pro Black</family>
                <pointsize>12</pointsize>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="plainText">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="conversion_save">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Сохранить как текущую функцию</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="conversion_clear">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Очистить вывод</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="analyzeTab">
           <attribute name="title">
            <string>Анализ</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;В данной вкладке можно построить графики (вейформу) для логической функции.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Цвет фона:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="analayze_color_selector_1">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>Белый</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Черный</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Цвет графиков:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="analayze_color_selector_2">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>Синий</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Красный</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Зеленый</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Фиолетовый</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_analyze_generate">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Предпросмотр</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_analyze_save">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Сохранить в файл</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="graphScrollArea">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>744</width>
                 <height>389</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="analyze_graph">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <stylestrategy>PreferDefault</stylestrategy>
                   </font>
                  </property>
                  <property name="text">
                   <string>   Нажмите &quot;Предпросмотр&quot; для отрисовки графиков</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="schemeTab" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuScheme">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Схема</string>
    </property>
    <addaction name="action_scheme_open"/>
    <addaction name="action_scheme_save"/>
    <addaction name="action_scheme_compile"/>
    <addaction name="action_scheme_clear"/>
    <addaction name="action_scheme_quit"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuScheme"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>Новый проект</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить проект</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Документация</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionSource_code">
   <property name="text">
    <string>Исходный код</string>
   </property>
  </action>
  <action name="action_scheme_open">
   <property name="text">
    <string>Открыть схему</string>
   </property>
  </action>
  <action name="action_scheme_save">
   <property name="text">
    <string>Сохранить схему</string>
   </property>
  </action>
  <action name="action_scheme_compile">
   <property name="text">
    <string>Скомпилировать схему</string>
   </property>
  </action>
  <action name="action_scheme_quit">
   <property name="text">
    <string>Закрыть редактор</string>
   </property>
  </action>
  <action name="action_scheme_clear">
   <property name="text">
    <string>Очистить всё</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
